web-ui/app/js/mail_view/ui/mail_view.js

```diff
@@ -96,72 +96,79 @@ define(
       };
 
       this.checkEncrypted = function(mail) {
-        if(_.isEmpty(mail.security_casing.locks)) {
-          return {
-            cssClass: 'security-status__label--not-encrypted',
-            label: 'not-encrypted'
-          };
-        }
-
-        var statusClass = ['security-status__label--encrypted'];
+        var statusClass = ['security-status__label'];
         var statusLabel;
+        var tooltip;
 
-        var hasAnyEncryptionInfo = _.any(mail.security_casing.locks, function (lock) {
-          return lock.state === 'valid';
-        });
-
-        if(hasAnyEncryptionInfo) {
-          statusLabel = 'encrypted';
+        if(_.isEmpty(mail.security_casing.locks)) {
+          statusClass.push('--not-encrypted');
+          statusLabel = 'not-encrypted';
+          tooltip = 'This message is not encrypted.';
         } else {
-          statusClass.push('--with-error');
-          statusLabel = 'encryption-error';
+          statusClass.push('--encrypted');
+
+          var hasAnyEncryptionInfo = _.any(mail.security_casing.locks, function (lock) {
+            return lock.state === 'valid';
+          });
+
+          if(hasAnyEncryptionInfo) {
+            statusLabel = 'encrypted';
+            tooltip = 'This message was encrypted.';
+          } else {
+            statusClass.push('--with-error');
+            statusLabel = 'encryption-error';
+            tooltip = 'This message had an encryption error.';
```

Comment:
@anikarni We changed the functions based in your suggestion, could you please take a look? 😄